linux_os/guide/system/accounts/accounts-banners/gui_login_banner/dconf_gnome_banner_enabled/oval/shared.xml

```diff
@@ -27,8 +27,8 @@
   </ind:textfilecontent54_test>
   <ind:textfilecontent54_object id="obj_banner_gui_enabled"
   version="1">
-    <ind:path>/etc/dconf/db/gdm.d/</ind:path>
-    <ind:filename operation="pattern match">^.*$</ind:filename>
+    <ind:path>/var/lib/gdm/.config/dconf</ind:path>
```

Comment:
@ggbecker why this change? If I can configure `/etc` is the top of the tree, that should be checked over `/var`.

Reply:
I'm not sure if I understood what you are trying to say. But files under `/etc/dconf/db/gdm.d/` serve only the purpose of input configuration for `dconf`. If someone changes the database by other means, this configuration won't reflect to files under `/etc/dconf/db/gdm.d/`. That's why we are changing dconf to use text backend which synchronizes the binary database to the file `/var/lib/gdm/.config/dconf/user.txt` and that's why we are changing the check to this file.

Reply:
@ggbecker is correct, I will rephrase a bit: If (and only if) the configuration under `/etc/dconf/db/...` specifies BOTH the respective key values AND those values are configured to be locked, subsequent execution of `dconf` will make sure that this configuration is applied, and local user settings are ignored.
However, OVAL can't check in detail what `dconf` did, as it saves it's database to a binary file. We are able to check the DB if it is switched to a text format, but it may be that in that case, we lose the possibility of locking settings - see the conversation above.
In any way, the GDM settings are not user settings, so the respective database is  located in `/var/lib/gdm/.config/dconf/`, and if the backend is set to text keyfiles, it is the `/var/lib/gdm/.config/dconf/user.txt` file. We have verified this on a VM, and it indeed works like this.

Reply:
A couple of observations here:
1. The OVAL checks need to check the `/etc` configuration (static configuration) for the system administrator configuration in addition to the `/var/lib` (runtime configuration) checks. 
2. Having a Dconf probe is becoming more necessary.